File: kata-6kyu/vowels-back.js
function vowelBack(s){
  //             abcdefghijklmnopqrstuvwxyz
  const codes = 'vkbaafpqistuvwnyzabtpvfghi';
  return s.split('').map(el=>codes[el.charCodeAt(0)-97]).join('')
}

File: python-kata-6kyu/vowels-back.py
def vowel_back(st):
    abc = 'abcdefghijklmnopqrstuvwxyz' 
    codes = 'vkbaafpqistuvwnyzabtpvfghi'
    s = ''
    for i in st:
        s+= codes[abc.index(i)]
    return s
